packages/client/tests/functional/decimal/precision/tests.ts

```diff
@@ -62,5 +62,9 @@ testMatrix.setupTestSuite(
         mongo - decimals are not supported
     `,
     },
+    skipDriverAdapter: {
+      from: ['js_mssql'],
+      reason: `The mssql driver appears to be losing precision even when receiving a stringified decimal`,
+    },
```

Comment:
Fixing `DECIMAL` support in SQLServer might need follow-up work in a new PR, with lower priority than this one.
Jacek notes that decimals work fine "until a certain precision".